astro/src/content/blog/open-community-issues.mdx

```diff
@@ -0,0 +1,31 @@
+ 
+In addition to the open source projects just mentioned, there is an [issues]( https://github.com/FusionAuth/fusionauth-issues) open source project for the core product as well. It also allows customers and those interested to have an open and transparent conversation about any issues they may experience with FusionAuth and give the community an opportunity to vote on how important the issue is to them.  It also allows another avenue of support for those on the [Community Plan](https://fusionauth.io/pricing?step=plan&hosting=self-hosting). The Community Plan is free and support is available through the community. Github is one of those community support vehicles. If you are experiencing an issue with your instance, in addition to searching the [Forum](https://fusionauth.io/community/forum/) and [Slack](https://join.slack.com/t/fusionauth/shared_invite/zt-6ykfov9u-MTwA1A6ptv0Gr6KcFBVG4w) channels, you can search the [issues](https://github.com/FusionAuth/fusionauth-issues/issues).  You can search both the open and closed issues.  If the issue is closed, you can find the reason for your issue and hopefully a fix. If there is an existing issue, you can upvote\downvote an issue by giving the thumbs up or thumbs down as a reaction to help FusionAuth prioritize the issue. If the issue does not exist, you can create a new issue. The best part about this is the FusionAuth developers monitor this project. Not only do they monitor, they create issues in this project when they uncover issues during their development. By simply reporting issues, you are able to actively participate in the development of the product. 
+
+## What is the best way to create an issue? 
+
+The best way to create an issue is to be as informative as possible. If you believe it is a bug, provide detailed steps to reproduce the issue. The more detailed you are, the easier it will be for the developers to see what you are seeing and fix it. 
+
+## Is every issue fixed? 
+
+Not every issue is a bug. In addition to bugs there are multiple types of issues such as documentation, performance, enhancement and others. However, once an issue has been triaged by the development team, it is addressed accordingly. Just because the software does not behave as you would expect, does not mean it is a defect. There can be multiple ways to implement a feature and at times, the community will submit what they see as an issue to be later told that the software is working as expected. This may not be the desired outcome for the reporter, but they now have the information they need to move forward. If they need more information, the reporter can also see any discussion and reasoning that has taken place in the comments of the issue because they are open source.
```

Comment:
```suggestion
Not every issue is a bug. In addition to bugs, there are multiple types of issues such as documentation, performance, enhancement, and others. However, once an issue has been triaged by the development team, it is addressed accordingly. Just because the software does not behave as you would expect, does not mean it is a defect. There can be multiple ways to implement a feature and at times, the community will submit what they see as an issue to be later told that the software is working as expected. This may not be the desired outcome for the reporter, but they now have the information they need to move forward. If they need more information, the reporter can also see any discussion and reasoning that has taken place in the comments of the issue because they are open source.
```